### PR TITLE
[lists-loops branch] replace string loops with lists

### PR DESCRIPTION
**Pull request by biokcb**
Removing all looping over strings in the loops episode of the lists-loops episode swap branch to avoid adding in an extra concept to discuss that might not add enough value for learners. Going to read over this again to check for typos and clarity before marking it ready for review (set as a draft for now). Let me know if there's additional changes that could be made. Thanks!

In reference to #496

### CODE REVIEW

File: _episodes/03-loop.md [hunk 1]
```diff
@@ -192,33 +192,34 @@ of the loop body (e.g. `end for`); what is indented after the `for` statement be
...
+for value in names:
     length = length + 1
-print('There are', length, 'vowels')
+print('There are', length, 'names in the list.')
 ~~~
 {: .language-python}
 
 ~~~
-There are 5 vowels
+There are 3 elements in the list.
```
Comment (maxim-belkin) [inline _episodes/03-loop.md]: `elements` or `names` or ...?

Comment (biokcb) [reply _episodes/03-loop.md]: Yep, thanks for catching that, it's supposed to be names :)

**Comment by biokcb** [comment]
Ok, I've read through it again and fixed anything else that popped up as weird/as a typo. Would appreciate feedback on whether or not the exercise I used is okay or if there is a better suggestion ("Summing a list"). Others I thought of would need to introduce the concept of creating an empty list (which, at this point, I'm wondering why it isn't mentioned at all) or having them essentially do what `enumurate` does but explicitly. Thanks!

**Comment by annefou** [comment]
I wonder if removing the last example "Reverse a String" is needed. I find it quite useful. Any reason why you removed it?

**Comment by biokcb** [comment]
@annefou Hi, we discussed removing iteration over strings as a concept overall in #496. Since lists are introduced before looping now, we can loop over them instead of strings. To keep the string iteration, we have to also introduce this as a concept and it does not add much utility in terms of understanding the concept of looping or usefulness in most projects. This I believe will help reduce overall cognitive load as well. I had a tough time coming up with a replacement problem for this one, so if you have suggestions on a better one that would be just as useful, that would be awesome! 

I also just realized there is another string based looping problem in a later lesson that should probably also be replaced. Any thoughts?

**Comment by maxim-belkin** [comment]
@biokcb, let's move the exercise on reversing a string to `_extras/extra_exercises.md`.

**Comment by annefou** [comment]
Very good suggestion @maxim-belkin 
Thanks.

**Comment by maxim-belkin** [comment]
@biokcb @annefou, I added that exercise to the "Extra" page.

**Comment by annefou** [comment]
Thanks @maxim-belkin

**Comment by maxim-belkin** [comment]
Thanks, Anne! And thank you, Kristen, for you contribution!

**Pull request merged**